docs/data/introduction/support/support.md

```diff
@@ -1,28 +1,37 @@
 # Support
 
-<p class="description">How to get support for MUI X components, including feature requests, bug fixes, answers to how-to questions, and technical support from the team.</p>
+<p class="description">Learn how to get support for MUI X components; including feature requests, bug fixes, answers to how-to questions, and technical support from the team.</p>
 
 ## GitHub
 
 We use GitHub issues as a bug and feature request tracker.
-If you think you've found a bug, or you have a new feature idea, please start by [making sure it hasn't already been reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).
-You can search through existing issues and pull requests to see if someone has reported one similar to yours.
 
-[Open an issue](https://github.com/mui/mui-x/issues/new/choose) in the MUI X repo.
+If you think you've found a bug, or you have a new feature idea:
+
+1. Please start by [making sure it hasn't already been reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).
+   You can search through existing issues and pull requests to see if someone has reported one similar to yours.
+2. Then, if no duplicates exists, [open an issue](https://github.com/mui/mui-x/issues/new/choose) in the MUI X repository.
 
 ## Stack Overflow
 
-Visit Stack Overflow to ask questions and read crowdsourced answers from expert developers in the MUI community, as well as MUI maintainers.
+We use Stack Overflow for how-to questions. Answers are crowdsourced from expert developers in the MUI community as well as MUI maintainers.
+
+You can search through existing questions and answers to see if someone has asked a similar question using one of [these tags](https://stackoverflow.com/questions/tagged/mui-x+or+mui-x-charts+or+mui-x-data-grid+or+mui-x-date-picker):
+
+- mui-x
+- mui-x-data-grid
+- mui-x-date-picker
+- mui-x-charts
 
-[Post a question about MUI X](https://stackoverflow.com/questions/tagged/mui-x) on Stack Overflow using the "mui-x" tag.
+If you can't find your answer, [ask a new question](https://stackoverflow.com/questions/ask?tags=reactjs%20mui-x) using the relevant tags.
 
 ## Technical support
 
 :::info
 The technical support covers only MUI X components.
 :::
 
-When purchasing a MUI X Pro or Premium license you get access to technical support until the end of your subscription.
+When purchasing a MUI X Pro or Premium license you get access to technical support until the end of your active license.
```

Comment:
A change we are making with @rluzists1 we are taking steps to define this "active license" notion https://github.com/mui/mui-private/pull/332. Avoiding "subscription" as it has a bad-rep.